src/function/table/arrow_conversion.cpp

```diff
@@ -509,6 +509,11 @@ void ColumnArrowToDuckDB(Vector &vector, ArrowArray &array, ArrowScanLocalState
 			IntervalConversionMonths(vector, array, scan_state, nested_offset, size);
 			break;
 		}
+		case ArrowDateTimeType::MONTH_DAY_NANO: {
+			FlatVector::SetData(vector, (data_ptr_t)array.buffers[1] + GetTypeIdSize(vector.GetType().InternalType()) *
```

Comment:
Our interval type has microseconds, not nanoseconds, so directly using the interval type is unlikely to be correct. Could you add some tests for this using e.g. the Python client/integration?